smac/optimizer/smbo.py

```diff
@@ -153,9 +154,28 @@ def start(self):
         Detects whether we the optimization is restored from previous state.
         """
         self.stats.start_timing()
+
         # Initialization, depends on input
         if self.stats.ta_runs == 0 and self.incumbent is None:
-            self.incumbent = self.initial_design.run()
+            logging.info('Running initial design')
+            # Run the initial deisgn only if intensifying using SMAC's procedure
+            # SH or HB methods get initialized by 1 run with random configurations
+            if not isinstance(self.intensifier, SuccessiveHalving):
+                # Intensifier initialization
+                self.incumbent = self.initial_design.run()
+            else:
+                # SH/HB initialization - get default configuration + random configurations
```

Comment:
shouldn't that be handled inside of the initial design?
E.g., if we evaluate multiple configurations in the "old" initial design, we also use intensification to compare these.

Reply:
It is handled inside the initial design now. I had to modify the initial design to accommodate it though.